common/templates/ellipses.html

```diff
@@ -17,19 +17,19 @@
         </button>
     </div>
     <div ng-if="config.selectMode == multiSelect" class="chaise-checkbox">
-        <input type="checkbox" ng-checked="selected || selectDisabled" ng-click="onSelect()" ng-disabled="selectDisabled">
+        <input type="checkbox" ng-checked="selected" ng-click="onSelect()" ng-disabled="selectDisabled">
         <label />
     </div>
 </td>
-<td ng-repeat="val in rowValues track by $index">
+<td ng-if="config.showNull || tuple.uniqueId" ng-repeat="val in rowValues track by $index">
```

Comment:
What tuple would not have a unique ID?

Reply:
The value of the key can be null for faceting (for both scalar and entity), and therefore the uniqueId would be empty string. But I needed to differentiate between empty string and null. I know it's not ideal for uniqueId to be null, but I couldn't think of a better way to handle it.